tests/testthat/test-istr.R

```diff
@@ -180,7 +180,7 @@ test_that("grouped_by includes the type of scenario", {
 test_that("with type ipr, for each company and grouped_by value sums 1 (#216)", {
   .type <- "ipr"
   companies <- istr_companies |> slice(1)
-  scenarios <- istr_scenarios |> filter(type == .type)
+  scenarios <- xstr_scenarios |> filter(type == .type)
   inputs <- istr_inputs |> filter(type == .type)
 
   out <- istr(companies, scenarios, inputs)
```

Comment:
@maurolepore this test is failing without any good reason. The output of the second line should be`TRUE`. Please correct this and similar test! Thanks!
```
> c(na.omit(sum$value_sum))[1]
[1] 1
> c(na.omit(sum$value_sum))[1] == 1
[1] FALSE
```

Reply:
@maurolepore Thanks for resolving this issue! I learned something new